.eleventy.js

```diff
@@ -344,16 +344,24 @@ module.exports = function (eleventyConfig) {
       }
     }
 
+    const now = new Date();
     return [...monthly]
       .sort((a, b) => {
         return new Date(b[0]) - new Date(a[0]);
       })
       .map((month) => {
+        const absoluteDate = new Date(month[0]);
+        const isCurrentMonth = absoluteDate.getMonth() === now.getMonth() && absoluteDate.getFullYear() === now.getFullYear();
         return {
           date: new Date(month[0]).toLocaleDateString("en-us", {
             month: "long",
             year: "numeric",
           }),
+          absoluteDate: absoluteDate,
+          // current month is not stable because it is still updating
+          // RSS feed should not include the current month
+          // https://github.com/web-platform-dx/web-features-explorer/pull/23
+          isStableMonth: !isCurrentMonth,
```

Comment:
`isStableMonth` is for skipping current month in RSS feed.
I tried to implement it in nunjucks slice but could not figure out how to make it work.
https://mozilla.github.io/nunjucks/templating.html#slice